style.css

```diff
@@ -0,0 +1,101 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap");
+
+:root {
+    --primary-color: #A5B4FC;
+    --outline-color: #374151;
+    --dark-gray: #d3d5d8;
+    --light-gray: #eceff3 ;
+    --off-white: #f3f4f6;
+}
+
+body {
+    font-size: 1rem;
+    font-family: "Lato", Arial, Helvetica, sans-serif;
+    color: var(--outline-color) ;
+    background-color: var(--light-gray);
+    background-image: linear-gradient(to top left, var(--dark-gray),var(--light-gray));
+}
+
+.layout {
+    grid-template-areas:
+        "header header header header header"
+        "nav main main main main"
+        "nav main main main main"
+        "nav main main main main"
+        "footer footer footer footer footer";
+    display: grid;
+    gap: rem;
+}
+
+.nav {
+    grid-area: nav;
+    border: 1px ;
```

Comment:
Remove these random spaces before semicolon, present in few more lines

Reply:
done